website/content/en/docs/concepts/nodeclasses.md

```diff
@@ -551,6 +554,36 @@ spec:
 
 The `Custom` AMIFamily ships without any default `blockDeviceMappings`.
 
+## spec.instanceStorePolicy
+
+The `instanceStorePolicy` field controls how [instance-store](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html) volumes are handled. By default, Karpenter and Kubernetes will simply ignore them.
+
+### RAID0
+
+If you intend to use these volumes for faster node ephemeral-storage, set `instanceStorePolicy` to `RAID0`:
+
+```yaml
+spec:
+  instanceStorePolicy: RAID0
+```
+
+This will set the allocatable ephemeral-storage of each node to the total size of the instance-store volume(s). 
+
+The disks must be formatted & mounted in a RAID0 and be the underlying filesystem for the Kubelet & Containerd. Instructions for each AMI family are listed below:
+
+#### AL2
+
+On AL2, Karpenter automatically configures the disks through an additional boostrap argument (`--local-disks raid0`). The device name is `/dev/md/0` and its mount point is `/mnt/k8s-disks/0`. You should ensure any additional disk setup does not interfere with these.
+
+#### Others
+
+For all other AMI families, you must configure the disks yourself. Check out the [`setup-local-disks`](https://github.com/awslabs/amazon-eks-ami/blob/master/files/bin/setup-local-disks) script in [amazon-eks-ami](https://github.com/awslabs/amazon-eks-ami) to see how this is done for AL2.
+
+{{% alert title="Tip" color="secondary" %}}
+Since the Kubelet & Containerd will be using the instance-store filesystem, you may consider using a more minimal root volume size.
```

Comment:
Since this is the case, should we consider setting this more minimal root volume size for users by default? Is there a conservative value that we could set here that would cover the snapshot size for AL2, Bottlerocket, and other AMIs?

Reply:
Wasn't 100% sure on this without having the actual data to support it. Looks like AL2 is about 8 GiB so not all that much lower from the default `20GiB`. But if folks were previously bumping that up for ephemeral-storage and are now using instance-store, then that EBS volume would be unused.

Do you think that tip should instead say to use the default root volume size?

Reply:
I think the way that you have written it is fine. If we start hearing reports that we should be dropping down the default volume size after this goes in, we can always change it later.